src/commonComponents/Button/Button.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+import './styles.scss'
+
+function Button({name='Apply'}) {
+    return (
+        <button className="btn">{name}</button>
```

Comment:
className should also be able to apply passed prop. 
therefore it should be a backticked string which firstly has its own classname and after that adds passed classname